app/views/widgets/my_module/_widget_protocol.html.erb

```diff
@@ -1,2 +1,180 @@
 <div class="widget-content">
+  <div data-role="protocol-status-bar" style="display: inline;">
```

Comment:
I suggest you simply specify `protocol` here, and then use it (instead of `@protocol`) elsewhere in this file:
```
<% protocol = @my_module.protocol %>
```